quick-protobuf/examples/pb_rs_v3/data_types.proto

```diff
@@ -17,12 +17,12 @@ message FooMessage {
     uint32 f_uint32 = 3;
     uint64 f_uint64 = 4;
     sint32 f_sint32 = 5;
-    sint64 f_sint64 = 6 [default=4];
-    bool f_bool = 7 [default=true];
+    sint64 f_sint64 = 6;
+    bool f_bool = 7;
     FooEnum f_FooEnum = 8;
     fixed64 f_fixed64 = 9;
     sfixed64 f_sfixed64 = 10;
-    fixed32 f_fixed32 = 11 [default=0];
+    fixed32 f_fixed32 = 11;
```

Comment:
Are the changes to these test files strictly necessary?

Reply:
That file contains Proto3, which doesn't allow for custom defaults from what I know, hence I removed those

Reply:
Oh interesting, so you are saying that this proto file is invalid? Might be worth sending in a different PR if it is not too much hassle. (For bisect reasons)

Reply:
Yep, I've separated these out for a new PR after this one (well, technically #247) goes through!